From foo@baz Wed Jun  3 09:47:57 PDT 2009
Date: Wed, 03 Jun 2009 09:47:57 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: pata_rdc: use PCI_DEVICE

Use the PCI_DEVICE macro, that's what it is there for...

Cc: Kevin Huang  <Kevin.Huang@rdc.com.tw>
Cc: Tomy Wang <Tomy.Wang@rdc.com.tw>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/pata_rdc/pata_rdc.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/staging/pata_rdc/pata_rdc.c
+++ b/drivers/staging/pata_rdc/pata_rdc.c
@@ -32,8 +32,8 @@
 
 
 static const struct pci_device_id rdc_pata_id_table[] = {
-	{ 0x17F3, 0x1011, PCI_ANY_ID, PCI_ANY_ID, 0, 0, RDC_17F31011},
-	{ 0x17F3, 0x1012, PCI_ANY_ID, PCI_ANY_ID, 0, 0, RDC_17F31012},
+	{ PCI_DEVICE(0x17F3, 0x1011), RDC_17F31011},
+	{ PCI_DEVICE(0x17F3, 0x1012), RDC_17F31012},
 	{ }	/* terminate list */
 };
 MODULE_DEVICE_TABLE(pci, rdc_pata_id_table);
